package.json

```diff
@@ -1,42 +1,46 @@
 {
-  "title": "Template for creating shield study addons",
   "name": "shield-studies-addon-template",
-  "version": "0.0.1",
   "description": "A basic add-on",
-  "main": "lib/index.js",
+  "version": "0.0.1",
   "author": "Gregg Lind <glind@mozilla.com>",
-  "license": "MIT",
   "bugs": {
     "url": "https://github.com/gregglind/shield-studies-addon-template/issues"
   },
-  "engines": {
-    "firefox": ">=38.0a1",
-    "fennec": ">=38.0a1"
-  },
-  "keywords": [
-    "jetpack",
-    "shield-study"
-  ],
   "dependencies": {},
   "devDependencies": {
+    "addons-linter": "^0.15.3",
     "chai": "^3.5.0",
     "grunt": "^1.0.1",
     "grunt-babel": "^6.0.0",
     "grunt-cli": "^1.2.0",
-    "grunt-eslint": "^18.1.0",
+    "grunt-eslint": "^19.0.0",
     "grunt-istanbul": "^0.7.0",
     "grunt-shell": "^1.3.0",
     "istanbul-jpm": "^0.1.0",
     "jpm": "^1.0.7",
     "shield-studies-addon-utils": "2.0.0",
+    "shield-study-cli": "1.1.2",
```

Comment:
Added this as a devDep so hopefully we don't have to install it globally. Global modules are weird and lame.